test/e2e/components/vl-action-group.js

```diff
@@ -1,10 +1,35 @@
 const { VlElement } = require('vl-ui-core').Test;
+const { VlButton } = require('vl-ui-button').Test;
+const { By } = require('selenium-webdriver');
 
 class VlActionGroup extends VlElement {  
-    constructor(driver, selector) {
-        super(driver, selector);
+    async getAlignedType() {
+        return this.getAttribute('align');
     }
 
+    async isLeftAligned() {
+        return (await this.getAlignedType()) === '';
```

Comment:
da weet gij toch niet dat niks left betekent?

Reply:
Wij hebben de implementatie gedaan, wij weten dat geen align attribuut of een leeg links betekend en gecoverd is in unit testen (voor zo ver die de echte stijl dekt).